resource/interface/client/main.lua

```diff
@@ -1,3 +1,5 @@
+---@alias IconProp 'fas' | 'far' | 'fal' | 'fat' | 'fad' | 'fab' | 'fak' | 'fass'
```

Comment:
Only `fas`, `far`, and `fab` are loaded into UI.

Reply:
Without the others it will throw a warning to those who use fontawesome pro (like me).

Reply:
They are also all the IconPrefixes defined in https://github.com/FortAwesome/Font-Awesome/blob/6.x/js-packages/%40fortawesome/fontawesome-common-types/index.d.ts

Reply:
Yeah that's true, you're right.